tests/Cargo.toml

```diff
@@ -23,6 +23,10 @@ harness = true
 name = "wgpu-gpu"
 harness = false
 
+[[test]]
+name = "wgpu-include-spirv"
+harness = true
+
```

Comment:
Don't see the need for a new test binary here, this can all go directly into the wgpu crate. You already have `SPIRV` & `SPIRV_RAW` variables there - I'd put those along above test methods into a tests sub-module in the wgpu crate as we do with most regular tests.

If for some reason we decided this should be separate after all, `docs/testing.md` would need updating as well.